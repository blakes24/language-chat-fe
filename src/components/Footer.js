import { makeStyles } from "@material-ui/core";

const useStyles = makeStyles((theme) => ({
  root: {
    backgroundColor: theme.palette.secondary.main,
    borderTop: "1px solid #E7E7E7",
    textAlign: "center",
    padding: "20px",
    left: "0",
    bottom: "0",
    minHeight: "60px",
    width: "100%",
    marginTop: "2rem",
    zIndex: 1201,
    position: "relative",
    display: "flex",
    alignItems: "center",
    justifyContent: "center",
    color: "white",
  },
  link: {
    color: theme.palette.secondary.light,
  },
}));

function Footer() {
  const classes = useStyles();
  return (
    <footer className={classes.root}>
      <div>
        <p>&#169; LangChat</p>
        <a className={classes.link} href="http://www.freepik.com">
          Graphics designed by stories/Freepik
        </a>
      </div>
    </footer>
  );
}

export default Footer;
