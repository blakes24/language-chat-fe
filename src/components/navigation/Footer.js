import { useStyles } from "./FooterStyles";

function Footer() {
  const classes = useStyles();
  return (
    <footer className={classes.root}>
      <div>
        <p>&#169; LangChat</p>
        <a className={classes.link} href="http://www.freepik.com">
          Graphics designed by stories/Freepik
        </a>
      </div>
    </footer>
  );
}

export default Footer;
