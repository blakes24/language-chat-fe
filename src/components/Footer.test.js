import { render } from "@testing-library/react";
import Footer from "./Footer";

test("renders without crashing", () => {
  render(<Footer />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
