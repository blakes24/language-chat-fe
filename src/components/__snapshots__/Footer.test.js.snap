// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="makeStyles-root-3"
  >
    <div>
      <p>
        Â© LangChat
      </p>
      <a
        class="makeStyles-link-4"
        href="http://www.freepik.com"
      >
        Graphics designed by stories/Freepik
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
